nss-day-cohort-48/rare-client-boastful-bulldogs
# Added Post Components

Added files for posts, added fetch calls, added a nav link and routing for posts.

## Steps to Test
1. git fetch --all
2. git checkout anm-post-list
3. serve and run debugger for server
4. verify there's a new link in the nav and that both links to you to different routes
5. also verify that 2 posts show up for all posts and my posts
